wallet/txauthor/go.mod

```diff
@@ -6,5 +6,5 @@ require (
 	github.com/btcsuite/btcd v0.22.0-beta.0.20220316175102-8d5c75c28923
 	github.com/btcsuite/btcd/btcutil v1.1.1
 	github.com/btcsuite/btcwallet/wallet/txrules v1.2.0
-	github.com/btcsuite/btcwallet/wallet/txsizes v1.1.0
+	github.com/btcsuite/btcwallet/wallet/txsizes v1.2.1
```

Comment:
I think this will also result in a change in `go.sum`, you might need to run `go mod tidy` in the `wallet/txauthor` directory.

Reply:
done